src/site/stages/build/process-cms-exports/schemas/output/node-health_care_region_page.js

```diff
@@ -1,5 +1,4 @@
 const { partialSchema } = require('../../transformers/helpers');
-const personProfileSchema = require('./node-person_profile');
```

Comment:
Why is this being removed?

Reply:
Leadership field is no longer in use on health_care_region_page in cms - has been removed, @jhonnyoddball cc: @kevwalsh